### ISSUE TITLE
Enterocutaneous fistula

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6420]
**Preferred term label:**
Enterocutaneous fistula

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0100819

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/entry/612567
PMID: 19890111  1/2

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
